grpc/shared_conn.go

```diff
@@ -192,6 +192,7 @@ func (sc *SharedConn) PeerConn() *webrtc.PeerConnection {
 	select {
 	case <-readyCh:
 	case <-failCh:
+		ret = nil
```

Comment:
Not exactly an exercised path. But as I was debugging something, I noticed that this might be the right thing to do?